packages/std/src/math/decimal.rs

```diff
@@ -478,6 +488,18 @@ mod tests {
     use super::*;
     use crate::{from_slice, to_vec};
 
+    #[test]
+    fn decimal_new() {
+        let expected = Uint128::from(300u128);
+        assert_eq!(expected, Decimal::new(expected).0);
```

Comment:
Can we use expected on the right for consistency?

```suggestion
        assert_eq!(Decimal::new(expected).0, expected);
```